tests/resources/rendered_templates/info.html

```diff
@@ -3,28 +3,29 @@
         text-align: left !important;
     }
 </style>
-
-<h3>Summary</h3>
-The preloaded data covers num_predictions predictions over num_entities entities from the dates start_date to end_date.
-<table>
-    <thead>
+<div style="width: 100%; max-width: 1400px;">
+    <h3>Summary</h3>
+    The preloaded data covers num_predictions predictions over num_entities entities from the dates start_date to end_date.
+    <table>
+        <thead>
+            <tr>
+            <th>Dataframe Name</th>
+            <th>Rows</th>
+            <th>Columns</th>
+            <th>Content</th>
+            </tr>
+        </thead>
         <tr>
-        <th>Dataframe Name</th>
-        <th>Rows</th>
-        <th>Columns</th>
-        <th>Content</th>
+        <td><code>sg.dataframe</code></td>
+        <td>1</td>
+        <td>1</td>
+        <td>Scores, features, configured demographics, and merged events for each prediction</td>
         </tr>
-    </thead>
-    <tr>
-    <td><code>sg.dataframe</code></td>
-    <td>1</td>
-    <td>1</td>
-    <td>Scores, features, configured demographics, and merged events for each prediction</td>
-    </tr>
-</table>
-<h4>Plot Functions</h4>
-<ul>
-    <li><code>sm.model_evaluation()</code> - Overall performance across thresholds</li>
-    <li><code>sm.cohort_evaluation(cohort_group)</code> - Performance split by specified cohort</li>
-    <li><code>sm.plot_outcome(outcome, intervention, cohort)</code> - Compare trends of interventions to outcomes</li>
-</ul> 
+    </table>
+    <h4>Plot Functions</h4>
+    <ul>
+        <li><code>sm.model_evaluation()</code> - Overall performance across thresholds</li>
```

Comment:
Not your PR, but are these even what we want to call out anymore? or should the refs be replaced ExploreXXX() classes?

Reply:
yeah, just fixing this to 

``` html
        <li><code>sm.ExploreModelEvaluation()</code> - Overall performance across thresholds</li>
        <li><code>sm.ExploreCohortEvaluation()</code> - Performance split by specified cohort</li>
        <li><code>sm.ExploreCohortOutcomeInterventionTimes()</code> - Compare trends of interventions to outcomes</li>
```